_posts/2022-02-05-frenchaward.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "An Open Source Software Award for scikit-learn"
+date: January 8, 2022
+categories:
+  - Events
+tags:
+  - Open Source
+  - Award
+  - Community
+  - French government
+featured-image: Frenchaward.png
+
+postauthors:
+  - name: François Goupil
+    website: https://www.linkedin.com/in/fran%C3%A7ois-goupil/
+    image: francois_goupil.jpeg 
+---
+
+<div>
+  <img src="/blog/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+We are pleased to announce that scikit-learn has received a prize for open-source scientific software from the French government. It is great recognition for all the community of contributors and users of a project born in France. Congratulation to the worldwide community for this great achievement!
+
+__A Community Award__ 
+scikit-learn was awarded for its very active community with more than 500,000 users per month and 2,200 contributors. Scikit-learn prides itself on being able to showcase its best practices for community building, an essential element of successful open-source software and open science innovation. Congratulations to all the projects and the teams that received the open-source software and open science award today. This work is inspiring for all of us!
+
+__The reaction of the Community__
+
+>“I literally owe my career in the data space to scikit-learn. It’s not just a framework but a school of thought regarding predictive modeling. Super well deserved, folks :) “ 
+Maykon Schots from Brasil
+
+>“Well done everyone for getting us here :)” 
+Joel Nothman from Australia
+
+__About the Award__
+
+For the first year, the Ministry of Higher Education, Research and Innovation awarded the Open Science Prizes for Free Research Softwares. Ten software developed by French teams were rewarded for their contribution to the advancement of scientific knowledge.
```

Comment:
@francoisgoupil 
Can you add a few links to these items?  Thank you.

Reply:
The link below the article is a reference this section. I think it is enough